osgar/tools/calibfish.py

```diff
@@ -0,0 +1,84 @@
+import cv2
```

Comment:
Maybe add some description in the header, please. My coworkers then use __doc__ as argsparse argument, which is maybe also option?